ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2

```diff
@@ -54,13 +58,15 @@
 {%             else %}
 {%                 set switch.platform = l3leaf.defaults.platform %}
 {%             endif %}
-{%             if bgp_as is defined and bgp_as is not none and overlay_routing_protocol|lower == 'ibgp' %}
-{%                  set leaf.bgp_as = bgp_as %}
+{%             if bgp_as is arista.avd.defined and switch.overlay_routing_protocol == 'ibgp' %}
```

Comment:
add `| lower` to allow various input of ibgp / iBGP etc...

`if bgp_as is arista.avd.defined and switch.overlay_routing_protocol | lower == 'ibgp' `